### ISSUE TITLE
You have exceeded your request quota for this API

### ISSUE THREAD

**Issue by 112617** [gmplot/gmplot#88]
I can use this code before, but I got trouble with "Google Maps JavaScript API warning: NoApiKey" these days because Google changed their policy. How can I fix this? / Where shall I correct?

**Comment by vgm64** [comment]
You can pass an apikey to GoogleMapPlotter when you create the object. You should be able to sign up at [developers.google.com](https://developers.google.com/maps/documentation/javascript/get-api-key).

**Comment by 112617** [comment]
> You can pass an apikey to GoogleMapPlotter when you create the object. You should be able to sign up at [developers.google.com](https://developers.google.com/maps/documentation/javascript/get-api-key).

Thanks ! I'll have a try :)

**Comment by hlongmore** [comment]
You can pass the apikey in, but the API url used for `geocode` and thus also `from_geocode` does not make use of the apikey:
```
        geocode = requests.get(
            'http://maps.googleapis.com/maps/api/geocode/json?address="%s"' % location_string)
```

**Comment by frslm** [comment]
Fixed in #111.
